src/system.c

```diff
@@ -1675,6 +1675,38 @@ void SySetGapRootPath( const Char * string )
 
 #endif
 
+/****************************************************************************
+**
+*F syLongjmp( <jump buffer>, <value>)
+** Perform a long jump
+**
+*F SignalSyLongjmp( <func> )
+** Register a function to be called before longjmp is called.
```

Comment:
That's not what this names conveys to me. How about `RegisterLongjmpHandler` or `RegisterLongjmpSignalHandler` or `RegisterLongjmpObserver` or so? It should match the final name of `SignalBreakloop`, I guess